The following model is an exsisting model that was created by me, I have load the model and showing the accuracy of the model on the test dataset of mnist digit recognition dataset.

## Loading The Model

In [1]:
from keras.models import model_from_json

json_file = open('final_model.json', 'r')
loaded_model_json = json_file.read()
json_file.close()
loaded_model = model_from_json(loaded_model_json)
# load weights into new model
loaded_model.load_weights("final_model.h5")

## Model Summary

In [2]:
loaded_model.summary()

Model: "sequential_2"
_________________________________________________________________
Layer (type)                 Output Shape              Param #   
conv2d_7 (Conv2D)            (None, 28, 28, 32)        320       
_________________________________________________________________
conv2d_8 (Conv2D)            (None, 28, 28, 32)        9248      
_________________________________________________________________
batch_normalization_4 (Batch (None, 28, 28, 32)        128       
_________________________________________________________________
max_pooling2d_4 (MaxPooling2 (None, 14, 14, 32)        0         
_________________________________________________________________
conv2d_9 (Conv2D)            (None, 14, 14, 128)       36992     
_________________________________________________________________
conv2d_10 (Conv2D)           (None, 14, 14, 128)       147584    
_________________________________________________________________
batch_normalization_5 (Batch (None, 14, 14, 128)      

## Loading testX and testY For Evaluation

In [3]:
from keras.datasets import mnist
from keras.utils import to_categorical

(trainX, trainY), (testX, testY) = mnist.load_data()
trainX = trainX.reshape((trainX.shape[0], 28, 28, 1))
testX = testX.reshape((testX.shape[0], 28, 28, 1))
trainY = to_categorical(trainY)
testY = to_categorical(testY)

trainX = trainX.astype('float32')
testX = testX.astype('float32')

trainX = trainX / 255.0
testX = testX / 255.0

## Compiling The Model

In [4]:
loaded_model.compile(loss='categorical_crossentropy', metrics=['accuracy'],optimizer='adam')

## Evaluating The Accuracy

In [5]:
_, acc = loaded_model.evaluate(testX, testY, verbose=0)

acc

0.9926999807357788

## Conclusion

By the exsisting model named 'final_model.h5', we are able the achieve an accuracy of **99.27%**.